ceph-pr-clang-tidy/config/ceph-pr-clang-tidy.yml

```diff
@@ -0,0 +1,61 @@
+- job:
+    name: ceph-pr-clang-tidy
+    node: small
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph: Clang-tidy checks'
+    concurrent: true
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: 15
+          artifact-days-to-keep: 15
+      - github:
+          url: https://github.com/ceph/ceph/
+
+  parameters:
+  - string:
+      name: ghprbPullId
```

Comment:
I suspect this is probably provided by the ghprb plugin, and doesn't need to/shouldn't be a settable parameter

Reply:
What's the response to this?

Reply:
I removed `ghprbPullId` as a settable parameter. However in others Jenkins jobs it is set as a settable parameter.

Reference: 
1. 
https://github.com/ceph/ceph-build/blob/e8d18c7aef15fa0424d096ba73a6e0d701a5215a/ceph-pull-requests-arm64/config/definitions/ceph-pull-requests-arm64.yml#L13-L16

2. 
https://github.com/ceph/ceph-build/blob/e8d18c7aef15fa0424d096ba73a6e0d701a5215a/ceph-perf-pull-requests/config/definitions/ceph-perf-pull-requests.yml#L137-L140

3. 
https://github.com/ceph/ceph-build/blob/e8d18c7aef15fa0424d096ba73a6e0d701a5215a/ceph-pull-requests/config/definitions/ceph-pull-requests.yml#L27-L30

Reply:
@dmick and what about this?

Reply:
We can leave it in for now; I don't think it's necessary but it might be useful for testing

Reply:
@dmick I have added  `node: jammy && small
`